projects/pgai/pgai/cli.py

```diff
@@ -411,3 +442,15 @@ def cli():
 vectorizer.add_command(vectorizer_worker)
 vectorizer.add_command(download_models)
 cli.add_command(vectorizer)
+
+
+@cli.command()
+@click.option(
+    "-d",
+    "--db-url",
+    type=click.STRING,
+    default="postgres://postgres@localhost:5432/postgres",
+    show_default=True,
+)
```

Comment:
should we have an `--if-not-exists` option?

Reply:
done